app/models/authentication_request_header.rb

```diff
@@ -0,0 +1,14 @@
+class AuthenticationRequestHeader < Authentication
+
+  def ansible_config_format
```

Comment:
Naming wise, this implies that any ansible thing will have this config format.  Is that a true statement?  If not, then either the name needs a change, or it needs to live elsewhere.